docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sqlite_data.md

```diff
@@ -23,33 +26,25 @@ In this guide we will demonstrate how to connect Great Expectations to data in a
 
 ### 1. Import GX and instantiate a Data Context
 
-```python title="Python code"
-import great_expectations as gx
-
-context = gx.get_context()
-```
+<ImportGxAndInstantiateADataContext />
 
 ### 2. Determine your connection string
 
 For this example we will use a connection string to connect to our PostgreSQL database.  In PostgreSQL, connection strings are formatted like:
 
-```python title="Python code"
-my_connection_string = "sqlite:///<PATH_TO_DB_FILE>"
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py connection_string"
 ```
 
 ### 3. Create a SQLite Datasource
 
 Creating a PostgreSQL Datasource is as simple as providing the `add_sqlite(...)` method a `name` by which to reference it in the future and the `connection_string` with which to access it.
 
-```python title="Python code"
-datasource_name = "my_datasource"
-my_connection_string = "sqlite:///<PATH_TO_DB_FILE>"
```

Comment:
@TrangPham same situation: document and snippet show different things (we should not change this, because the document was already "approved").  Thanks.